css/catalog.css

```diff
@@ -43,14 +43,164 @@
     gap: 8px;
 }
   
+.category {
+    width: 320px;
+    padding: 24px;
+    gap: 16px;
+    background-color: #1D1C28;
+    box-sizing: border-box;
+}
+
+.category__header {
+    width: 272px;
+    border-bottom: 1px solid rgba(191, 197, 238, 0.12);
+    padding: 16px;
+    font-family: 'Bakbar One', Arial, sans-serif;
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 32px;
+}
+
+.category__options {
+    width: 320px;
+    padding: 24px;
+    gap: 16px;
+    display: flex;
+    flex-direction: column;
+}
+
+.checkbox {
+    appearance: none;
+    background-color: #0D0D15;
+    border-radius: 4px;
+    position: absolute;
+    cursor: pointer;
+    width: 14px;
+    height: 14px;
+    border: 2px solid rgba(212, 217, 242, 0.26);
+    position: relative;
+    display: inline-block;
+    margin-right: 10px;
+    transition: background .3s;
+}
+
+.checkbox:checked {
+    background-color: #0265DC;
+    border-color: #0265DC;
+}
+
+.checkbox:checked:after {
+    content: '';
+    color: white;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+}
+
+.options {
+    position: relative;
+}
+
+.radio {
+    appearance: none;
+    background-color: #0D0D15;
+    border: 2px solid rgba(212, 217, 242, 0.26);
+    border-radius: 50%;
+    position: absolute;
+    cursor: pointer;
+    width: 14px;
+    height: 14px;
+    position: absolute;
```

Comment:
Double position: absolute; 109 and 113 lines

Reply:
removed

Reply:
fixed